hwdb.d/60-sensor.hwdb

```diff
@@ -309,6 +309,10 @@ sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrY13D_KB133.103:b
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo*:pnC3W6_AP108_4GB:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
+# Chuwi Ubook X (CWI535)
```

Comment:
I didn't write `Ubook X N4100` here because the fix for the N4100 and the 2023 (i5-1020Y) versions might be the same.